packages/cardano-services/src/Program/programs/providerServer.ts

```diff
@@ -231,6 +231,7 @@ export const loadProviderServer = async (
       },
       logger
     );
+  logger.fatal(args);
```

Comment:
:eyes:

Reply:
Was this committed by accident?

Reply:
Yep, development change refuse.
Even if your comment wasn't so specific, here I had no questions as the suggestion is clear enough as it is 😃

Reply:
removed the refuse with cba27e3cf